ai-ml/jupyterhub/data.tf

```diff
@@ -0,0 +1,37 @@
+data "aws_eks_cluster_auth" "this" {
+  name = module.eks.cluster_name
+}
+
+data "aws_availability_zones" "available" {}
+
+data "aws_acm_certificate" "issued" {
+  domain   = var.acm_certificate_domain
+  statuses = ["ISSUED"]
+}
+
+data "kubernetes_service" "elb" {
+  metadata {
+    name = "proxy-public"
+    namespace = var.name
+  }
+  depends_on = [module.kubernetes_data_addons]
+}
+data "aws_ecrpublic_authorization_token" "token" {
+  provider = aws.ecr
+}
+data "aws_caller_identity" "current" {}
+data "aws_eks_addon_version" "this" {
```

Comment:
remove when CNI is created under the addons module

Reply:
done